### ISSUE TITLE
Invoice QR code wrong scale

### ISSUE THREAD

**Issue by TomasHo** [BlueWallet/BlueWallet#203]
LN invoice in a iPad Mini creates a QR code that does not fit on to the screen(vertically) a cannot be scanned. Unable to zoom out.
Screenshots below.
![img_5747](https://user-images.githubusercontent.com/12121219/50647682-83e82780-0f79-11e9-818d-36a6bc6e7ce0.jpg)
![img_5746](https://user-images.githubusercontent.com/12121219/50647683-83e82780-0f79-11e9-855c-7a712d51b4f8.jpg)

**Comment by Overtorment** [comment]
Reproduces if you launch the app in album orientation. Works ok if you kill the app and launch in portrait orientation.
@marcosrdz we should just forbid album orientation. Should be specified somewhere in xcode settings..?

**Comment by klauslovgreen** [comment]
Yes it kind of works if relaunching in portrait mode but the QR is still huge (edge to edge) and if running to landscape it get stuck like above.

**Comment by Overtorment** [comment]
Big QR is a must as it packs  quite long invoice string, and low-end devices with bad cameras will read it better.
There should be some padding tho

**Comment by klauslovgreen** [comment]
Can't even see thew full QR code and there is no text address to copy
